website/content/en/docs/concepts/nodepools.md

```diff
@@ -92,7 +92,6 @@ spec:
       # These are all optional and provide support for additional customization and use cases.
```

Comment:
Can you make sure this update is made in the `/preview` path as well? This is how we are actually carrying docs changes forward when we release a new minor version

Reply:
@jonathan-innis done